examples/LossMinimization/loss_minimization.jl

```diff
@@ -0,0 +1,117 @@
+using Distributions
+using LinearAlgebra
+using Random
+using Test
+using Plots
+using CalibrateEmulateSample.EKP
```

Comment:
Renamed to `CalibrateEmulateSample.EnsembleKalmanProcesses` in newest master

Reply:
and renamed!